presets.js

```diff
@@ -0,0 +1,416 @@
+	{ label: 'zone 2 power off', id: 'Z2OFF', act: 'power', cat: 'Power' },
+	{ label: 'zone 3 power off', id: 'Z3OFF', act: 'power', cat: 'Power' },
+]
+const PRESETS_VOL = [
+	//VOLUME needs to be moved to its own area (or figure out the expansion problem)
+	{ label: 'main zone volume up', id: 'MVUP', act: 'volume', cat: 'Volume' },
+	{ label: 'main zone volume down', id: 'MVUDOWN', act: 'volume', cat: 'Volume' },
+	{ label: 'zone 2 volume up', id: 'Z2UP', act: 'volume', cat: 'Volume' },
+	{ label: 'zone 2 volume down', id: 'Z2DOWN', act: 'volume', cat: 'Volume' },
+	{ label: 'zone 3 volume up', id: 'Z3UP', act: 'volume', cat: 'Volume' },
+	{ label: 'zone 3 volume down', id: 'Z3DOWN', act: 'volume', cat: 'Volume' },
+]
+const PRESETS_SURROUND = [
+	{ label: 'Surround mode Music', id: 'MSMUSIC', act: 'surround', cat: 'Surround' },
+	{ label: 'Surround mode Game', id: 'MSGAME', act: 'surround', cat: 'Surround' },
+	{ label: 'Surround mode Direct', id: 'MSDIRECT', act: 'surround', cat: 'Surround' },
+	{ label: 'Surround mode Pure Direct', id: 'MSPURE DIRECT', act: 'surround', cat: 'Surround' },
+	{ label: 'Surround mode Stereo', id: 'MSSTEREO', act: 'surround', cat: 'Surround' },
+	{ label: 'Surround mode Auto', id: 'MSAUTO', act: 'surround', cat: 'Surround' },
+	{ label: 'Surround mode Dolby Digital', id: 'MSDOLBY DIGITAL', act: 'surround', cat: 'Surround' },
+	{ label: 'Surround mode DTS', id: 'MSDTS SURROUND', act: 'surround', cat: 'Surround' },
+]
+const PRESETS_SETVOL = [
+	{ 
+        label: 'set volume zone 1', 
+        actID: 'set_volume', 
+        opt1: 'zone', 
+        val1: 'MV', 
+        opt2: 'set_volume', 
+        val2: '00', 
+        cat: 'Volume',	
+    },
+	{
+		label: 'set volume zone 2',
+		actID: 'set_volume',
+		opt1: 'zone',
+		val1: 'Z2',
+		opt2: 'set_volume',
+		val2: '00',
+		cat: 'Volume',
+	},
+	{
+		label: 'set volume zone 3',
+		actID: 'set_volume',
+		opt1: 'zone',
+		val1: 'Z3',
+		opt2: 'set_volume',
+		val2: '00',
+		cat: 'Volume',
+	},
+]
+const PRESETS_NAVIGATE = [
+	{ label: 'up', id: 'MNCUP', act: 'navigate', cat: 'Menu navigation' },
+	{ label: 'down', id: 'MNCDN', act: 'navigate', cat: 'Menu navigation' },
+	{ label: 'left', id: 'MNCLT', act: 'navigate', cat: 'Menu navigation' },
+	{ label: 'right', id: 'MNCRT', act: 'navigate', cat: 'Menu navigation' },
+	{ label: 'enter', id: 'MNENT', act: 'navigate', cat: 'Menu navigation' },
+	{ label: 'return', id: 'MNRTN', act: 'navigate', cat: 'Menu navigation' },
+	{ label: 'option', id: 'MNOPT', act: 'navigate', cat: 'Menu navigation' },
+	{ label: 'info', id: 'MNINF', act: 'navigate', cat: 'Menu navigation' },
+	{ label: 'setup menu on', id: 'MNMEN ON', act: 'navigate', cat: 'Menu navigation' },
+	{ label: 'setup menu off', id: 'MNMEN OFF', act: 'navigate', cat: 'Menu navigation' },
+]
+
+module.exports = {
+	initPresets: function () {
+		const presets = {}
+
+		for (let x in PRESETS_SETVOL) {
```

Comment:
by the way you constantly using `PRESETS_SETVOL[x]`, I assume that it would be better to use a for...of loop

Reply:
I will keep these in mind for future work.. this is a learning area for me.